### PR TITLE
merge to 4.5.14 onPrem installation doc prep

### PR DESCRIPTION
**Pull request by lilachmaliniak**
@noam-arad changes were merged

### CODE REVIEW

File: media-server_deployment.md [hunk 1]
```diff
@@ -1,27 +1,41 @@
-# Kaltura Server #
+## Machine prerequisites:
+ - Centos 6.8+
```
Comment (kalturaguy) [inline media-server_deployment.md]: Or ubuntu...

File: media-server_deployment.md [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
+ - WowzaStreamingEngine 4.6.0
```
Comment (kalturaguy) [inline media-server_deployment.md]: 4.5.0+

Comment (lilachmaliniak) [inline media-server_deployment.md]: done

File: media-server_deployment.md [hunk 3]
```diff
@@ -1,27 +1,41 @@
...
 - Add Wowza to plugins.ini.
 
-## Configuration ##
+
+## Configuration:
 - Add the IP range containing the Wowza machine IP to the @APP_DIR@/configurations/local.ini:  
 	internal_ip_range = {required range}  
   Note that this is not necessary for a Hybrid eCDN installation.
 - Edit the @APP_DIR@/configurations/broadcast.ini file according to the broadcast.template.ini file.
 - If there is a need for non-default configuration of the WSE (for instance, different port), you will need to create a custom configuration file on your API machine under @APP_DIR@/configurations/media_servers.ini, according to the template found here: https://github.com/kaltura/media-server/blob/3.0.8/media_servers.template.ini.
```
Comment (kalturaguy) [inline media-server_deployment.md]: we don't need /media_servers.ini anymore

File: media-server_deployment.md (hunk 1 above)
Comment (lilachmaliniak) [inline media-server_deployment.md]: done

File: media-server_deployment.md [hunk 4]
```diff
@@ -1,27 +1,41 @@
...
  - FEATURE_LIVE_STREAM_RECORD
  - FEATURE_KALTURA_LIVE_STREAM
  - FEATURE_KALTURA_LIVE_STREAM_TRANSCODE
 
 
-
-## Edge Servers: ##
-media_servers.ini is optional and needed only for custom configurations.
+## Edge Servers:
+- media_servers.ini is optional and needed only for custom configurations.
```
Comment (kalturaguy) [inline media-server_deployment.md]: no need for this

File: media-server_deployment.md (hunk 2 above)
Comment (lilachmaliniak) [inline media-server_deployment.md]: done

File: media-server_deployment.md [hunk 5]
```diff
@@ -1,27 +1,41 @@
...
+## Additional libraries:
+- commons-codec-1.4.jar
+- commons-httpclient-3.1.jar
+- commons-logging-1.1.1.jar
+- commons-lang-2.6.jar
+
 
-## Plugins: ##
+## Plugins:
 - Add Wowza to plugins.ini.
```
Comment (kalturaguy) [inline media-server_deployment.md]: don't know what this is

File: media-server_deployment.md (hunk 3 above)
Comment (lilachmaliniak) [inline media-server_deployment.md]: removed

File: media-server_deployment.md [hunk 6]
```diff
@@ -1,27 +1,41 @@
...
+ - WowzaStreamingEngine 4.6.0
+ - Java jre 1.7.
+ - kaltura group (gid = 613) or any other group that apache user is associated with.
+ - Write access to @WEB_DIR@/content/recorded directory.
+ - Read access to symbolic link of @WEB_DIR@/content under @WEB_DIR@/content/recorded:
+  ln –s @WEB_DIR@/content @WEB_DIR@/content/recorded/content
 
 
+## Additional libraries:
+- commons-codec-1.4.jar
```
Comment (kalturaguy) [inline media-server_deployment.md]: this is brough with gradle, we can remove this

File: media-server_deployment.md (hunk 3 above)
Comment (lilachmaliniak) [inline media-server_deployment.md]: updated

File: media-server_deployment.md (hunk 4 above)
Comment (lilachmaliniak) [inline media-server_deployment.md]: removed

File: media-server_deployment.md (hunk 5 above)
Comment (lilachmaliniak) [inline media-server_deployment.md]: removed

Comment (lilachmaliniak) [inline media-server_deployment.md]: added mention of plugin.ini in BE configuration section

File: media-server_deployment.md (hunk 6 above)
Comment (lilachmaliniak) [inline media-server_deployment.md]: done

**Pull request merged**